skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/auth/AuthorizeAnsattSakTilgang.kt

```diff
@@ -37,10 +40,12 @@ fun AuthorizeAnsattSakTilgang(
             val navIdent = principal.navIdent
 
             val sakDeferred = async { penService.hentSak(call, saksId) }
+
             val enheterDeferred = async { navansattService.hentNavAnsattEnhetListe(call, navIdent) }
 
             val ikkeTilgang = sakDeferred.await().map { sak ->
                 call.attributes.put(sakKey, sak)
+                call.attributes.put(saktypeKey, sak.sakType)  // Benyttes av PdlService for å velge behandlingsnummer
```

Comment:
Blir  det litt for "magisk" å overføre noe som er helt nødvendig til PdlService på denne måten?

Reply:
ja , det ble litt magi , derfor jeg la til kommentar i tilfelle noen lurer på hva greia er , men er ikke supert nei. Samtidig har jeg ikke så lyst til å drive å hente pen sak mange ganger....hmmmm

Reply:
ok